tock/hours/forms.py

```diff
@@ -148,22 +156,21 @@ class TimecardObjectForm(forms.ModelForm):
 
     class Meta:
         model = TimecardObject
-        fields = ['project', 'hours_spent', 'notes']
+        fields = ['project', 'hours_spent', 'notes', 'project_allocation']
 
     def clean_project(self):
         data = self.cleaned_data.get('project')
-
         try:
             data = Project.objects.get(id=data)
         except Project.DoesNotExist:
             raise forms.ValidationError('Invalid Project Selected')
-
         return data
 
     def clean_hours_spent(self):
         return self.cleaned_data.get('hours_spent') or 0
 
     def clean(self):
+        # print(self.cleaned_data)
```

Comment:
**nitpick:** I prefer to remove debugging prints rather than commenting them out, just to keep things tidy.